packages/transition-backend/src/services/simulation/methods/AccessibilityMapSimulation.ts

```diff
@@ -8,23 +8,20 @@ import random from 'random';
 import { EventEmitter } from 'events';
 import { point as turfPoint } from '@turf/turf';
 
-import { SimulationAlgorithmDescriptor } from 'transition-common/lib/services/simulation/SimulationAlgorithm';
 import { SimulationRunDataAttributes } from 'transition-common/lib/services/simulation/SimulationRun';
 import { TransitRoutingBaseAttributes } from 'chaire-lib-common/lib/services/routing/types';
 import { SimulationMethodFactory, SimulationMethod } from './SimulationMethod';
-import dataSourceDbQueries from 'chaire-lib-backend/lib/models/db/dataSources.db.queries';
 import placesDbQueries from '../../../models/db/places.db.queries';
 import nodesDbQueries from '../../../models/db/transitNodes.db.queries';
 import { TransitAccessibilityMapCalculator } from '../../accessibilityMap/TransitAccessibilityMapCalculator';
 import TransitAccessibilityMapRouting from 'transition-common/lib/services/accessibilityMap/TransitAccessibilityMapRouting';
 import NodeCollection from 'transition-common/lib/services/nodes/NodeCollection';
+import {
+    AccessMapSimulationDescriptor,
```

Comment:
If you want to rename Accessibility by Access, I think you should add some readme somewhere explaining that it is an abbreviation. The term "access" is already used for "access/egress" segments from origin to the first node and from the last node to the destination. I would suggest to use the full Accessibility name instead here for this reason.